src/libserver/registry/HorseRegistry.cpp

```diff
@@ -235,4 +262,34 @@ void HorseRegistry::GiveHorseRandomPotential(
     randomDist(_randomDevice));
 }
 
+const SkinInfo* HorseRegistry::GetSkinInfo(data::Tid skinTid) const
+{
+  auto it = _skins.find(skinTid);
+  if (it != _skins.end())
+  {
+    return &it->second;
+  }
+  return nullptr;
+}
+
+int32_t HorseRegistry::GetManeColorGroup(data::Tid maneTid) const
+{
+  auto it = _manes.find(maneTid);
+  if (it != _manes.end())
+  {
+    return it->second.colorGroup;
+  }
+  return 0;
+}
```

Comment:
When you apply previous comments, you should be able to do something like this:
```suggestion
int32_t HorseRegistry::GetManeColorGroup(data::Tid maneTid) const
{
  auto it = _manes.find(maneTid);
  if (it == _manes.end())
    throw std::runtime_error("Mane does not exist");
  return it->second.colorGroupId;
}
```